### PR TITLE
Add community section

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/dashboard/ha-config-dashboard.ts [hunk 1]
```diff
@@ -134,6 +141,53 @@ class HaConfigDashboard extends LitElement {
...
+                "discord",
+                html`<a
+                  href="https://www.home-assistant.io/join-chat/"
+                  target="_blank"
+                  rel="noreferrer"
+                  >Chat</a
+                >`,
+                "blog",
+                html`<a
+                  href="https://www.home-assistant.io/blog/"
```
Comment (ludeeus) [inline src/panels/config/dashboard/ha-config-dashboard.ts:173]: Should we use the documentationUrl helper here, so you get to beta blog if you are on beta?

File: src/panels/config/dashboard/ha-config-dashboard.ts [hunk 2]
```diff
@@ -134,6 +141,53 @@ class HaConfigDashboard extends LitElement {
...
+                  >Blog</a
+                >`,
+                "newsletter",
+                html`
+                  <a
+                    href="https://www.home-assistant.io/newsletter/"
+                    target="_blank"
+                    rel="noreferrer"
+                    >Newsletter</a
+                  ></span
```
Comment (ludeeus) [inline src/panels/config/dashboard/ha-config-dashboard.ts:185]: These kinds of splits, while correct does not look that great :smile:

File: src/panels/config/dashboard/ha-config-dashboard.ts (hunk 1 above)
Comment (zsarnett) [reply src/panels/config/dashboard/ha-config-dashboard.ts:173]: There is a beta blog?

Comment (balloob) [reply src/panels/config/dashboard/ha-config-dashboard.ts:173]: Yeah, while beta is active, the release blog post is visible on rc.home-assistant.io. The documentationUrl helper handles this automatically.

File: src/panels/config/dashboard/ha-config-dashboard.ts (hunk 2 above)
Comment (zsarnett) [reply src/panels/config/dashboard/ha-config-dashboard.ts:185]: ah its orphaned

**Review by balloob — APPROVED** [review]

**Pull request merged**